### ISSUE TITLE
Displacement count and Lightmapped_4WayBlend

### ISSUE THREAD

**Issue by spaztron64** [Facepunch/garrysmod-requests#1437]
The 5500 displacement number bump is a good addition, but what's stopping you from reaching something like 8192, or even 32768 like CS:GO?
Additionally, will support for the 4-way blend shader be added?

**Comment by Kefta** [comment]
Read Rubat's comment on https://github.com/Facepunch/garrysmod-requests/issues/1257.

**Comment by Sherlock2004** [comment]
Lightmapped_4WayBlend will never appear in garry's mod???

**Comment by VCSoldier** [comment]
any chance you could comment on if this will ever be added, since we never really got an answer and anytime someone makes a seperate request for it you redirect them to this one @robotboy655 

personally i feel like this would allow mappers to have much more to work with in regards to mapping, and it wouuld also fix a bunch of csgo textures so people can finally use the 3 currently unusable blend channels.

**Comment by Sherlock2004** [comment]
> any chance you could comment on if 4wayblend will ever be added, since we never really got an answer and anytime someone makes a seperate request for it you redirect them to this one [@robotboy655](https://github.com/robotboy655)
> 
> personally i feel like this would allow mappers to have much more to work with in regards to mapping, and it wouuld also fix a bunch of csgo textures so people can finally use the 3 currently unusable blend channels.

CSGO uses a more advanced shader version - MultiBlend. But it seems that even if Gmod at least has Lightmapped_4WayBlend, it will be possible to redirect calls to it and the maps will really look better, and mappers will get more freedom.

**Comment by bruhhhxyt** [comment]
> > Rubat doesn't do anything, bruh
> 
> You're being presumptuous and myopic in this particular statement of yours

Yes, I don't like when someone doesn't give a reason on why they don't do it and just closes the duplicates

**Comment by robotboy655** [comment]
I have added shader support for `Lightmapped_4WayBlend`, but no tool support yet.
